test/jenkins_tests/run_multi_node_tests.sh

```diff
@@ -40,3 +40,31 @@ python $ROOT_DIR/multi_node_docker_test.py \
     --mem-size=60G \
     --shm-size=60G \
     --test-script=/ray/test/jenkins_tests/multi_node_tests/large_memory_test.py
+
+# Test that the example applications run.
+
+# docker run --shm-size=10G --memory=10G $DOCKER_SHA \
+#     python /ray/examples/lbfgs/driver.py
+
+# docker run --shm-size=10G --memory=10G $DOCKER_SHA \
+#     python /ray/examples/rl_pong/driver.py \
+#     --iterations=3
+
+# docker run --shm-size=10G --memory=10G $DOCKER_SHA \
+#     python /ray/examples/hyperopt/hyperopt_simple.py
+
+# docker run --shm-size=10G --memory=10G $DOCKER_SHA \
+#     python /ray/examples/hyperopt/hyperopt_adaptive.py
+
+docker run --shm-size=10G --memory=10G $DOCKER_SHA \
+    python /ray/python/ray/rllib/a3c/example.py \
+    --environment=PongDeterministic-v0 \
+    --iterations=2
+
+# docker run --shm-size=10G --memory=10G $DOCKER_SHA \
+#     python /ray/python/ray/rllib/policy_gradient/example.py \
+#     --iterations=2
+
+docker run --shm-size=10G --memory=10G $DOCKER_SHA \
+    python /ray/python/ray/rllib/evolution_strategies/example.py \
```

Comment:
Maybe set --env-name as well here? We could also run rllib/train.py instead, which might supercede the example.py files.

Otherwise this looks good to me.